treeherder/webapp/api/performance_data.py

```diff
@@ -1337,6 +1348,223 @@ def _get_signatures_map(signatures, grouped_values, option_collection_map):
 
         return signatures_map, header_names, platforms
 
+    """
+    _process_new_stats does the following for base and new:
+    1. calculate basic statistics, mean, median, variance, standard deviation, average, min, max
+    2. Runs a Shapiro-Wilk normality test, is it normal and interpretation
+    3. Runs a Kolmogorov-Smirnov test for goodness of fit, is fit good and interpretation
+    4. Runs a Mann-Whitney U test test null hypothesis, data between two base and new, calculates p-value
+    5. Calculate Cliff's delta and interpretation, check if new is better than base based on delta and lower_is_better
+    6. Interpret Common Language Effect Size, with p-value and p-threshold (.05), calculate statistical effect and significance with what level of confidence interval
+    7. Estimate out KDE with Silverman bandwidth, check if multimodal or irregular, calc interpretation if regressed, improved, or neutral
+    8. Plot KDE with ISJ bandwidth to reduce smoothing
+
+    return object:
+
+    {
+            "base_rev": { name, data, header, title, info_link, sample_count, mean, median, variance, standard_deviation, min, max },
+            "new_rev": new_rev_info,
+            "shapiro_wilk_test": {
+                "test_name": "Shapiro-Wilk",
+                "shapiro_stat_base",
+                "pvalue_base",
+                "interpretation_base",
+                "is_normal_base",
+                "shapiro_stat_new",
+                "pvalue_new",
+                "interpretation_new",
+                "is_normal_new",
+            },
+            "ks_test": {
+                "test_name": "Kolmogorov-Smirnov",
+                "ks_stat": ks_stat,
+                "pvalue": ks_p,
+                "comment": ks_comment,
+            },
+            "ks_warning": ks_warning,
+            "mann_whitney_test": { "test_name": "Mann-Whitney U", "statistic": mann_stat, "p_value": mann_pvalue, "interpretation": interpretation_mann },
+            "base_median": base_median,
+            "new_median": new_median,
+            "delta_value": delta_value,
+            "delta_percentage": delta_percentage,
+            "mann_pvalue": mann_pvalue,
+            "cliffs_delta": cliffs_delta,
+            "cliffs_interpretation": cliffs_interpretation,
+            "is_fit_good": is_fit_good,
+            "is_new_better": is_new_better,
+            "is_regression": is_regression,
+            "is_improvement": is_improvement,
+            "more_runs_are_needed": more_runs_are_needed,
+            "direction_of_change": direction,  # 'neutral', 'better', or 'worse'
+            "dke_isj_plot": {
+                "bandwidth": "Improved Sheather-Jones",
+                "kernel": "Gaussian",
+                "title": "Distribution Comparison Kernel Density Estimate (KDE)",
+                "xlabel": "Value",
+                "ylabel": "Density",
```

Comment:
I wouldn't include these values in the response.

Reply:
I have reopened this, in case you think it's helpful for the UI to keep them in the response